docs/DeveloperGuide.adoc

```diff
@@ -316,7 +375,7 @@ a chart.
 |- More work is required.
 |===
 
-//tag::Report[]
+[#Report]
```

Comment:
I don't think there is a need to change this, there is an ending tag too, it's used for `includes`

Reply:
I referred to line 378 from 256